src/main/java/uk/gov/pay/directdebit/webhook/sandbox/resources/WebhookSandboxResource.java

```diff
@@ -1,36 +1,60 @@
 package uk.gov.pay.directdebit.webhook.sandbox.resources;
 
-import java.util.List;
-import javax.inject.Inject;
-import javax.ws.rs.POST;
-import javax.ws.rs.Path;
-import javax.ws.rs.core.Response;
-
 import com.codahale.metrics.annotation.Timed;
 import uk.gov.pay.directdebit.gatewayaccounts.model.PaymentProvider;
 import uk.gov.pay.directdebit.payments.model.PaymentState;
+import uk.gov.pay.directdebit.events.model.SandboxEvent;
 import uk.gov.pay.directdebit.payments.model.Transaction;
+import uk.gov.pay.directdebit.events.services.SandboxEventService;
 import uk.gov.pay.directdebit.payments.services.TransactionService;
 
+import javax.inject.Inject;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Response;
+import java.time.ZonedDateTime;
+
 import static javax.ws.rs.core.Response.Status.OK;
 
 @Path("/v1/webhooks/sandbox")
 public class WebhookSandboxResource {
 
     private final TransactionService transactionService;
+    private final SandboxEventService sandboxEventService;
 
     @Inject
-    public WebhookSandboxResource(TransactionService transactionService) {
+    public WebhookSandboxResource(TransactionService transactionService, SandboxEventService sandboxEventService) {
         this.transactionService = transactionService;
+        this.sandboxEventService = sandboxEventService;
+    }
+    
+    enum SandboxEventAction {
+        SANDBOX_EVENT_ACTION
```

Comment:
I think this (and the cause) should probably be `PAID_OUT` or something to reflect what’s already there

Reply:
Good idea I've updated.